var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGsutilFeedstock(PythonPackage):
```

Comment:
```suggestion
class PyGsutil(PythonPackage):
```
Will need to update file name too.